applications/DEMApplication/custom_constitutive/DEM_rolling_friction_model.h

```diff
@@ -49,13 +48,15 @@ namespace Kratos{
 
         virtual std::unique_ptr<DEMRollingFrictionModel> CloneUnique();
 
-        virtual void ComputeRollingFriction(SphericParticle* p_element, SphericParticle* p_neighbor, double LocalContactForce[3], array_1d<double, 3>& mContactMoment, double indentation);
+        virtual void InitializeSolutionStep() {}
+
+        virtual void ComputeRollingFriction(SphericParticle* p_element, SphericParticle* p_neighbor, double LocalCoordSystem_2[3], double LocalContactForce[3], array_1d<double, 3>& mContactMoment) {}
         
-        virtual void ComputeRollingFrictionWithWall(double LocalContactForce[3], SphericParticle* p_element, Condition* const wall, double indentation, array_1d<double, 3>& mContactMoment);
+        virtual void ComputeRollingFrictionWithWall(SphericParticle* p_element, Condition* const wall, double LocalCoordSystem_2[3], double LocalContactForce[3], array_1d<double, 3>& mContactMoment) {}
 
-        virtual void InitializeSolutionStep() {}
```

Comment:
why do yoy remove the InitiSolStep?

Reply:
Actually it was not removed, I just moved its position.